pros/cli/conductor.py

```diff
@@ -159,7 +159,7 @@ def upgrade(ctx: click.Context, project: c.Project, query: c.BaseTemplate, **kwa
     """
     analytics.send("upgrade-project")
     if not query.name:
-        for template in project.templates.keys():
+        for template in tuple(project.templates.keys()):
```

Comment:
out of curiosity what is the point of this change?

Reply:
The number of keys changes within the loop so we make a copy of the keys as a tuple to iterate over.